src/app/pages/home/home.component.html

```diff
@@ -16,6 +16,7 @@
       </div>
     </div>
 
-    <img class="intro__image" src="assets/icons/coding-illustration.svg">
+    <img class="intro__image" src="../../../assets/icons/coding-illustration.svg">
```

Comment:
Why did you add the `../../../` prefix?

Reply:
I think my code editor did this... there is no reason for this change. I will undo it.